spec/requests/create_dro_spec.rb

```diff
@@ -168,7 +173,7 @@
 
   context 'when the signed_id indicates a globus file' do
     let(:signed_id) { 'globus://abc123/file2.txt' }
```

Comment:
This one is actually necessary as it's used in the request above. I'm leaning towards a comment on here to avoid confusion rather than duplicating the request setup from above.

Reply:
🆗